cluster/gke_operations.go

```diff
@@ -0,0 +1,5 @@
+package cluster
+
+type OperationInfoer interface {
```

Comment:
This should be unexported, operation constructors should return struct pointer types instead.